test_requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Core project/common application testing requirements
 
+django-celery
```

Comment:
At one point I was trying to set up async Celery testing which I think required that library, however I went a different route and this can be removed